helm/fiftyone-teams-app/values.yaml

```
@@ -43,6 +43,9 @@ apiSettings:
   # -- Additional labels for the `teams-api` deployment. [Reference][labels-and-selectors].
   labels: {}
 
+  # -- Number of pods in the `teams-api` deployment's ReplicaSet.
+  # Must be used in conjunction with `apiSettings.env.FIFTYONE_SHARED_ROOT_DIR`.
```

Should we link to the helm/docs/configure-ha-teams-api.md doc the better explains this?

```suggestion
  # When > 1, you must also configure volumes, volumeMounts and set `apiSettings.env.FIFTYONE_SHARED_ROOT_DIR`.
  # For more information see [Reference][configure-ha-teams-api]
```

And then set the markdown link in helm/fiftyone-teams-app/README.md.gotmpl


```yaml
[configure-ha-teams-api]: https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/docs/configure-ha-teams-api.md
```

---

Resolved.